chains/ethereum/config_test.go

```diff
@@ -71,6 +72,70 @@ func TestParseChainConfig(t *testing.T) {
 	}
 }
 
+// TestParseChainConfigHex tests the parseChainConfig with hex values
+func TestParseChainConfigHex(t *testing.T) {
+	commonHexValue := "0x1234"
+	expectedHex, parseErr := utils.ParseUint256OrHex(&commonHexValue)
+	if parseErr != nil {
+		t.Fatalf("unable to parse value %s, %v", commonHexValue, parseErr)
+	}
+
+	input := core.ChainConfig{
+		Name:         "chain",
+		Id:           1,
+		Endpoint:     "endpoint",
+		From:         "0x0",
+		KeystorePath: "./keys",
+		Insecure:     false,
+		Opts: map[string]string{
+			"bridge":             "0x1234",
+			"erc20Handler":       "0x1234",
+			"erc721Handler":      "0x1234",
+			"genericHandler":     "0x1234",
+			"gasLimit":           commonHexValue,
+			"gasMultiplier":      "1",
+			"maxGasPrice":        commonHexValue,
+			"minGasPrice":        "0",
+			"http":               "true",
+			"startBlock":         "10",
+			"blockConfirmations": "50",
+			"egsApiKey":          "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx", // fake key
+			"egsSpeed":           "fast",
+		},
+	}
+
+	out, err := parseChainConfig(&input)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	expected := Config{
+		name:                   "chain",
+		id:                     1,
+		endpoint:               "endpoint",
+		from:                   "0x0",
+		keystorePath:           "./keys",
+		bridgeContract:         common.HexToAddress("0x1234"),
+		erc20HandlerContract:   common.HexToAddress("0x1234"),
+		erc721HandlerContract:  common.HexToAddress("0x1234"),
+		genericHandlerContract: common.HexToAddress("0x1234"),
+		gasLimit:               expectedHex,
+		maxGasPrice:            expectedHex,
+		minGasPrice:            big.NewInt(0),
```

Comment:
Add this as an `expectedHex` value to test all cases.

Reply:
What do you mean?

The rest of the params are just regular address types. Did you mean to replace the occurrence of `0x1234` with `commonHexValue`?

Reply:
Sorry, if I was a bit unclear.
As far as I can see you added that `minGasPrice` can also be hex, but it is tested with int instead of hex value
in hex config test.

Reply:
Got it 👍 

I've changed the unit test to use a hex value:

6e97291